src/adcm_pytest_plugin/docker_utils.py

```diff
@@ -378,21 +405,34 @@ def adcm_container(
         ip=None,
         volumes=None,
     ):
+        config = ContainerConfig(
+            image=image,  # keep from black
+            labels=labels,
+            remove=remove,
+            name=name,
+            tag=tag,
+            ip=ip,
+            volumes=volumes,
+        )
+        return self.adcm_container_from_config(config)
+
+    @allure.step("Run ADCM container")
```

Comment:
Why you decided to remove {image}:{tag}?

Reply:
New implementation has its own `allure.step` message.
I see no reason to keep decorator on deprecated function.
May suggest instead to remove new implementation decorator and use context manager.

Reply:
Yes. That's exactly what I was talking about. I want to see {config.image}:{config.tag} in the new step implementation

Reply:
Done.